lib/basket.rb

```diff
@@ -0,0 +1,23 @@
+class Basket
+
+  attr_reader :basket
+
+  def initialize
+    @basket = {}
+  end
+
+  def view
+    @basket.map do |key, value|
+      "#{key}, £#{value}"
+    end.join("\n")
+  end
+
+  def total
+    @basket.values.sum
+  end
+
+  def review_order
+    "#{view}\nTotal: £#{total}"
+  end
+
+end
```

Comment:
`view` and `total` methods could be stored as private methods as `review_order` makes use of both of them and it looks like they won't need to be called external outside of the class